dev/com.ibm.ws.security.saml.sso_fat/fat/src/com/ibm/ws/security/saml/fat/SPInitiated/BasicSolicitedSPInitiatedTests.java

```diff
@@ -75,6 +75,8 @@ public static void setupBeforeTest() throws Exception {
         // add any additional messages that you want the "start" to wait for
         // we should wait for any providers that this test requires
         extraMsgs = getDefaultSAMLStartMsgs();
+        // make sure that we wait for ssl to be up and ready during the original start and server restarts
+        extraMsgs.add("CWWKO0219I:.*defaultHttpEndpoint-ssl.*");
```

Comment:
there's a constant for this message

https://github.com/OpenLiberty/open-liberty/blob/integration/dev/com.ibm.ws.security.fat.common/src/com/ibm/ws/security/fat/common/MessageConstants.java#L18

Reply:
SAML doesn't pull in those message constants at the moment.
I can look at pulling those into SAMLMessageConstants as long as there are no conflicts...